### PR TITLE
Twilio 4.0.0

### PR DESCRIPTION
**Pull request by luluhoc**
## What

What changes are being made? (e.g. feature, bug, docs etc.)

## Why

Why are these changes necessary?

New version of Twilio client that updates jsonwebtoken with vulnerability.

## How

How were these changes implemented?

## Checklist

Have you done all of these things?

- [x] Documentation added
- [x] Tests
- [x] Typescript definitions updated
- [x] Ready to be merged

### CODE REVIEW

**Comment by luluhoc** [comment]
@rejvban You can do it as a pre-release as this is RC-5 of Twilio. Thanks

**Comment by luluhoc** [comment]
readme updated @rejvban

**Comment by luluhoc** [comment]
@rejvban updated to 4.0.0 official version can you merge?

File: package.json [hunk 1]
```diff
@@ -42,8 +42,12 @@
     "prepublishOnly": "pinst --disable",
     "postpublish": "pinst --enable"
   },
+  "engines": {
+    "npm": ">=8.0.0",
+    "node": ">=14.0.0"
+  },
```
Comment (lkaric) [inline package.json]:
```suggestion
  "engines": {
    "npm": "please-use-yarn",
    "yarn": ">=1.17.3",
    "node": ">=14.0.0"
  },
```

File: lib/utils/twilio.interface.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import { Twilio } from 'twilio';
+import type { ClientOpts } from 'twilio/dist/lib/base/BaseTwilio';
```
Comment (lkaric) [inline lib/utils/twilio.interface.ts]: I try to keep the `type` imports at the bottom of the import sections

**Review by lkaric — CHANGES_REQUESTED** [review]

**Comment by luluhoc** [comment]
@rejvban both resolved

**Review by lkaric — APPROVED** [review]

**Pull request merged**